archive/c/c/fibonacci.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <errno.h>
+#include <inttypes.h>
+
+/*
+ * This limit is chosen because fib(93) is the first number that is
+ * still small enough to fit into a 64 bit integer.
+ */
+#define LIMIT 93
```

Comment:
I like this! I'm also interested in potentially testing the input to see if it's a number as mentioned in #480. Should we be striving for more complete programs like this?

Reply:
Coincidentally, I do already check for the validity of the entered number (the 2nd if in the main function).

I think it's generally important to implement all necessary error checking here, just because this repo should showcase the cleanest possible implementation imo.

Reply:
I’ll have to brush up on my C! Haha I couldn’t tell if it was verifying number input or not (I.e. “djks”), and I didn’t test it beyond a normal case.

Reply:
Also, I agree. I just know that some situations call for relaxing the requirements (see Revers a String) due to the complexity of the complete solution. I don’t think that applies here.